Bundle/QueryBundle/Helper/QueryHelper.php

```diff
@@ -185,6 +190,14 @@ public function buildWithSubQuery($containerEntity, QueryBuilder $itemsQueryBuil
             }
         }
 
+        if (strpos($query, ':currentUser') !== false && is_object($this->getCurrentUser())) {
+            if (is_object($this->getCurrentUser())) {
+                $itemsQueryBuilder->setParameter('currentUser', $this->getCurrentUser()->getId());
+            } else {
+                throw new AccessDeniedException();
```

Comment:
When the client is not logged, wouldn't be better showing blank instead of throwing an exception?